#!/usr/bin/env python
PACKAGE = "jetracer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kp", int_t, 0, "p", 350, 0, 2000)
gen.add("ki", int_t, 0, "i", 120, 0, 2000)
gen.add("kd", int_t, 0, "d",   0, 0, 2000)
gen.add("servo_bias", int_t, 0, "servo_bias",   0, -500, 500)

exit(gen.generate(PACKAGE, "jetracer", "jetracer"))